.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
```

Comment:
Why don't we enable the [Container based infrastructure](http://docs.travis-ci.com/user/workers/container-based-infrastructure/)?